src/app/checkout/step-2/existingPaymentMethods/existingPaymentMethods.component.js

```diff
@@ -167,6 +167,7 @@ export default angular
       defaultPaymentType: '<',
       hidePaymentTypeOptions: '<',
       cartData: '<',
+      brandedCheckoutItem: '<',
```

Comment:
This seems really annoying to have to pass around 😞 Could we pass the `showCoverFees` bool around instead (or maybe stash it somewhere? In React this is called prop drilling and can be solved by using context. Idk if there's a good solution in AngularJS.

Reply:
So, there isn't really a need to pass around the boolean since it is in local storage. We could maybe put this branded checkout item also in local storage and keep it updated there instead. This object holds the frequency and amounts that I care about primarily. I think that would end up with more code, though.